tools/docker/tests/test_flows.sh

```diff
@@ -149,7 +149,7 @@ test_init() {
         err "start GW+Repeater failed, abort"
         exit 1
     }
-    AL_MAC=$(docker exec -it gateway ${installdir}/bin/beerocks_cli -c bml_conn_map | grep IRE_BRIDGE | awk '{print $5}' | cut -d ',' -f 1)
+    AL_MAC=$(docker exec -it gateway ${installdir}/bin/beerocks_cli -c bml_conn_map | grep IRE_BRIDGE| head -1 | awk '{print $5}' | cut -d ',' -f 1)
```

Comment:
I would replace this with an explicit `aa:bb:cc:dd:00:30` (or whatever the correct MAC address is). It may feel a little fragile, but this entire script heavily relies on the exact way that we set up containers that I don't think relying on a specific MAC address is too much to ask.

(nitpick)